internal/entities/user.go

```diff
@@ -1 +1,11 @@
 package entities
+
+type User struct {
+	ID         int    `json:"id"`
+	Username   string `json:"username"`
+	Password   string `json:"password"`
```

Comment:
Why not use the hyphen directive? Any particular reason?
```suggestion
	Password   string `json:"-"`
```